.github/workflows/tests-on-push.yml

```diff
@@ -22,7 +22,9 @@ jobs:
           curl -sSL https://install.python-poetry.org | python3 -
           poetry install
       - name: build docs
-        run: poetry run mkdocs build
+        run: poetry run mkdocs build --strict
+      - name: check external links in docs
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
This is quite cool, it checks if you have broken (external) links. Thanks to this, I found 2 dead links in my docs.

Unfortunately, I found no tool that could check internal links like `[dead link](./dsp-tools-excel2json.md#lists-sectionnnnn)`. It's true that PyCharm displays such dead links in the "Problems" section, but due to the many false positives, I might easily miss the true positives. So if you know a better tool, let me know!

Reply:
https://www.npmjs.com/package/markdown-link-validator looks promising. I've tried it out and it seems to work also for relative links.